include/nbl/asset/ICommandBuffer.h

```diff
@@ -213,34 +213,9 @@ class NBL_API ICommandBuffer
 
     // hm now i think having begin(), reset() and end() as command buffer API is a little weird
```

Comment:
amend the comment, its having an `ICPUCommandBuffer` thats weird, maybe we should have a rendergraph

Reply:
Resolved.